qemu/tests/seabios_order_once.py

```diff
@@ -0,0 +1,87 @@
+import os
+import re
+import logging
+
+from avocado.utils import process
+
+from virttest import error_context
+from virttest import env_process
+from virttest import utils_misc
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    KVM Seabios test:
+    [seabios]Boot guest with "-boot order/once" option
+    1) Start VM with sga bios
+    2) Check boot order before reboot
+    3) Reboot the VM
+    4) Check boot order after reboot
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def create_cdroms(cdrom_test):
+        """
+        Create 'test' cdrom with one file on it
+        """
+        logging.info("creating test cdrom")
+        process.run("dd if=/dev/urandom of=test bs=10M count=1")
+        process.run("mkisofs -o %s test" % cdrom_test)
+        process.run("rm -f test")
+
+    def cleanup_cdroms(cdrom_test):
+        """
+        Removes created cdrom
+        """
+        logging.info("cleaning up temp cdrom images")
+        os.remove(cdrom_test)
+
+    def boot_check(info):
+        """
+        boot info check
+        """
+        return re.search(info, vm.serial_console.get_stripped_output(), re.S)
+
+    cdrom_test = params["cdrom_test"]
+    create_cdroms(cdrom_test)
+    params["start_vm"] = "yes"
+    env_process.preprocess_vm(test, params, env, params.get("main_vm"))
+
+    error_context.context("Start VM with sga bios", logging.info)
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+
+    vm.pause()
+    # Disable nic device, boot fail from nic device except user model
+    if params['nettype'] != 'user':
+        for nic in vm.virtnet:
+            process.system("ifconfig %s down" % nic.ifname)
+    vm.resume()
+
+    timeout = int(params.get("login_timeout", 360))
```

Comment:
I think the param name should be `"boot_timeout"`, and the value would be `60`

Reply:
Updated! 60s is too short to get info. Change to 90s. Thanks!